client/js/array-util.js

```diff
@@ -0,0 +1,18 @@
+const getRange = function(fromNum, toNum) {
```

Comment:
Set your text editor to use 2-space tabs instead of 4-space tabs and to “convert indentations to spaces”; ask me if you have any trouble. This is key for readability, especially when you checked your code into a central repo that other devs will work off of.